src/main/java/seedu/duke/Storage.java

```diff
@@ -0,0 +1,100 @@
+package main.java.seedu.duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import main.java.seedu.duke.todo.Task;
+
+/**
+ * Represents a storage. Allow Duke to read and write to local disk.
+ */
+public class Storage {
+    private static final String DIRECTORY_PATH = "data";
+    private static final String FILE_PATH = DIRECTORY_PATH + "/duke.txt";
+    private TaskList taskList;
+
+    public Storage(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Reads files from local disk and print out the content.
+     */
+    public void readFile() {
+        // load the file first
+        File directory = new File(DIRECTORY_PATH);
+        if (!directory.exists()) {
```

Comment:
I think the method name should be a verb, so maybe change exists() to checkExist instead?